src/luger.erl

```diff
@@ -144,14 +145,46 @@ init_sink(State = #state{}, #syslog_udp_config{ host = Host,
                 syslog_udp_facility = Facility}.
 
 log(Priority, Channel, Message) ->
-    case whereis(?PROC_NAME) of
-        undefined -> {error, luger_not_running};
-        _ -> do_log(Priority, Channel, Message)
-    end.
+    log(Priority, Channel, Message, whereis(?PROC_NAME)).
 
-do_log(Priority, Channel, Message) ->
+log(_Priority, _Channel, _Message, undefined) ->
+    {error, luger_not_running};
+log(Priority, Channel, Message, _) ->
     [{state, State}] = ets:lookup(?TABLE, state),
 
+    Now = erlang:convert_time_unit(erlang:monotonic_time(), native, seconds),
```

Comment:
You can just use `erlang:monotonic_time/1` instead of converting.  Potentially faster.